pkgs/by-name/fa/fakeroot/package.nix

```
@@ -14,7 +14,7 @@
 }:
 
 stdenv.mkDerivation (finalAttrs: {
-  version = "1.37.1.2";
+  version = "1.37.1.2-unstable-2025-07-05";
```

```suggestion
  version = "1.37.1.2";
```

This does not need to be updated. (and it implicitly breaks the `src` FOD as `src.rev` depends on it)

---

*facepalm* you're right, and it keeps working because i didn't remove the hash

---

Here the commit in `src` is still `upstream/1.37.1.2` instead of for example `95ead7684c4d88ef0a13bb26ec16ad3a6984934d`. Only in the latter case, and when no tag is associated with the commit, do we add `-unstable-` to the version.